src/lib/create.c

```diff
@@ -301,7 +301,7 @@ hash_bn(pgp_hash_t *hash, BIGNUM *bignum)
 static int
 hash_key_material(const pgp_seckey_t *key, uint8_t *result)
 {
-    pgp_hash_t hash;
+    pgp_hash_t hash = {0};
```

Comment:
@catap Can you elaborate? AFAIK this is a perfectly acceptable way of initializing a structure to zeroes in C99.